doc/sphinx/source/quickstart/installation.rst

```diff
@@ -214,3 +232,11 @@ confirm that no errors are reported:
 .. code-block:: bash
 
     python setup.py test
+
+.. note::
```

Comment:
I would remove this note, as it's not correct, setup.py also installs dependencies etc. The only thing setup.py cannot do is install the extra development dependencies, so that's why we recommend pip.

Reply:
do you want to rephrase it yourself or should I just remove it? I find it useful keeping it for people that ask what the difference between the two is

Reply:
Maybe it would be best to just remove it: our installation instructions are already quite lengthy, turning them into a complete Python manual would make them very difficult to follow.

Reply:
I removed the note. :-)